app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -0,0 +1,123 @@
+class ManageIQ::Providers::Vmware::CloudManager::RefreshParser < ManageIQ::Providers::CloudManager::RefreshParser
+  def self.ems_inv_to_hashes(ems, options = nil)
+    new(ems, options).ems_inv_to_hashes
+  end
+
+  def initialize(ems, options = Config::Options.new)
+    @ems                 = ems
+    @data                = {}
+    @data_index          = {}
+    @options             = options
+
+    @connection          = ems.connect
+
+    @inv = Hash.new { |h, k| h[k] = [] }
```

Comment:
I might be a little OCD but can you line up the `=` and don't think the extra new lines help readability here
Also there is a lot more padding to the left of the `=` than is needed